docs/QuickStart-Tutorial.md

```diff
@@ -148,14 +148,14 @@ var gameType = new GraphQLObjectType({
   interfaces: [nodeInterface],
 });
 
-var hidingSpotType = new GraphQLObjectType({
+const hidingSpotType = new GraphQLObjectType({
   name: 'HidingSpot',
   description: 'A place where you might find treasure',
   fields: () => ({
     id: globalIdField('HidingSpot'),
     hasBeenChecked: {
       type: GraphQLBoolean,
-      description: 'True if this spot has already been checked for treasure',
```

Comment:
Sorry for this. I was doing a copy-paste from the example. https://github.com/relayjs/relay-examples/blob/master/relay-treasurehunt/data/schema.js#L91